app/models/concerns/percentage.rb

```diff
@@ -0,0 +1,29 @@
+module Percentage
```

Comment:
We have done away with the need for this concern with the new decorator approach haven't we.